Fabric.Authorization.Domain/Models/GranularPermission.cs

```diff
@@ -23,11 +29,11 @@ public class GranularPermission : ITrackable, IIdentifiable, ISoftDelete
 
         public string CreatedBy { get; set; }
 
-        public string ModifiedBy { get; set; }
+        public string ModifiedBy { get; set; }        
 
         public override string ToString()
         {
-            return $"{Target}";
+            return $"{Identifier}";
```

Comment:
Should we include IdentityProvider here too?

Reply:
If Id becomes a concatenation of subject id and identity provider then it will be included here